app/controllers/register_api/applications_controller.rb

```diff
@@ -2,6 +2,29 @@ module RegisterAPI
   class ApplicationsController < ActionController::API
     include ServiceAPIUserAuthentication
 
-    def index; end
+    def index
+      application_choices = get_recruited_applications(
+        recruitment_cycle_year: recruitment_cycle_year_param,
+      )
+      render json: { data: MultipleApplicationsPresenter.new(application_choices, api: RegisterAPI).as_json }
+    end
+
+  private
+
+    def get_recruited_applications(recruitment_cycle_year:)
+      ApplicationChoice.includes([:application_form])
+        .where(application_forms: { recruitment_cycle_year: recruitment_cycle_year }).references(:application_forms)
+        .where(status: ApplicationStateChange::STATES_VISIBLE_TO_REGISTER)
+        .where.not(recruited_at: nil)
+    end
+
+    def recruitment_cycle_year_param
+      year = params.fetch(:recruitment_cycle_year)
```

Comment:
We've seen in the vendor API today someone passing in a parameter that does not exist, which then throws a `KeyError` and a 500 https://sentry.io/organizations/dfe-bat/issues/2296340364/?project=1765973&referrer=slack
Can we instead handle this gracefully and return a 422 instead?